roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -391,6 +403,7 @@
     es_container_security_context: "{{ _es_containers.elasticsearch.securityContext if _es_containers is defined and 'elasticsearch' in _es_containers and 'securityContext' in _es_containers.elasticsearch else None }}"
     deploy_type: "{{ openshift_logging_elasticsearch_deployment_type }}"
     es_replicas: 1
+    basic_auth_passwd: "{{ _logging_metrics_proxy_passwd | b64decode }}"
```

Comment:
we cant leave this encoded here and decode it on our side?

Reply:
We could, but that means its encoded in the DC, base64 encoded again in the request.  We would need to double decode it during the check.  I put the decode here so it wasnt plan text in the log file when running the installer.  I wanted to avoid bzs that might come up.